docs/core/tools/dotnet.md

```diff
@@ -314,81 +223,57 @@ For more information about each tool, type `dotnet <tool-name> --help`.
 
 ## Examples
 
-Creates a new .NET Core console application:
-
-`dotnet new console`
-
-Restore dependencies for a given application:
+Create a new .NET Core console application:
 
-`dotnet restore`
-
-[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
+```dotnetcli
+dotnet new console
+```
 
 Build a project and its dependencies in a given directory:
 
-`dotnet build`
+```dotnetcli
+dotnet build
+```
 
-Run an application DLL, such as `myapp.dll`:
+Run an application:
 
-`dotnet myapp.dll`
+```dotnetcli
+dotnet myapp.dll
+```
 
 ## Environment variables
 
-# [.NET Core 2.1](#tab/netcore21)
-
-`DOTNET_PACKAGES`
-
-The global packages folder. If not set, it defaults to `~/.nuget/packages` on Unix or `%userprofile%\.nuget\packages` on Windows.
-
-`DOTNET_SERVICING`
-
-Specifies the location of the servicing index to use by the shared host when loading the runtime.
-
-`DOTNET_CLI_TELEMETRY_OPTOUT`
+- `DOTNET_ROOT`
 
-Specifies whether data about the .NET Core tools usage is collected and sent to Microsoft. Set to `true` to opt-out of the telemetry feature (values `true`, `1`, or `yes` accepted). Otherwise, set to `false` to opt into the telemetry features (values `false`, `0`, or `no` accepted). If not set, the default is `false` and the telemetry feature is active.
+  Specifies the location of the .NET Core SDK and runtimes if they are not installed in the default location. The default location on Windows is `C:\Program Files\dotnet`. The default location on Linux and macOS is `/usr/share/dotnet`.
```

Comment:
SDK is optional - so I would word it something like "the location of the .NET Core runtimes and optionally SDKs if they..."

Reply:
In fact `DOTNET_ROOT` is only used for running apps via generated executables (apphosts), the muxer `dotnet.exe` ignores this env. variable. So it really only applies to running apps through executables - has no effect on SDK lookup and `dotnet.exe` itself`.

Reply:
There's also `DOTNET_ROOT(x86)` which is used on 64bit OS when running 32bit executable (instead of `DOTNET_ROOT` which is the 64bit version that case).